pkg/util/fieldindex/register.go

```diff
@@ -71,6 +64,79 @@ func RegisterFieldIndexes(c cache.Cache) error {
 		if err = c.IndexField(&v1.PersistentVolumeClaim{}, IndexNameForOwnerRefUID, ownerIndexFunc); err != nil {
 			return
 		}
+		// pod name
+		if err = indexPodName(c); err != nil {
+			return
+		}
+		// job owner
+		if err = indexJob(c); err != nil {
+			return
+		}
+		// broadcastjob owner
+		if err = indexBroadcastCronJob(c); err != nil {
+			return
+		}
 	})
 	return err
 }
+
+func indexPodName(c cache.Cache) error {
```

Comment:
Why do we need to create a field index for pod name?

Reply:
That was the existing code. Just formatted it in different func

Reply:
Please rename to indexPodNodeName, which is different from creating an index for Pod name.

Reply:
Addressed in https://github.com/openkruise/kruise/pull/461/commits/5b783be22bff9600f7f0cb820dc55ef2b64998fd